### ISSUE TITLE
Core's modules not coming up after doing an opam upgrade

### ISSUE THREAD

**Issue by fxfactorial** [the-lambda-church/merlin#395]
I recently did `opam upgrade` on my OS X machine after which auto completions are not coming in correctly. For example, 

I do 

`open Core.Std` and I expect to have String.split available, but doing code completion only gives me the stdlib's string functions. I can access the split func via typing out the full module path, ie. Core.Std.String.split, but this is a regression. (Module aliases issues?)

To recap, everything was working correctly, I did opam upgrade and now completions are incorrect.

FWIW, I'm using company-mode.

**Comment by fxfactorial** [comment]
So after reinstalling core three times, merlin now works correctly.
